src/Webtoons/WebtoonsParser.ts

```diff
@@ -0,0 +1,380 @@
+import { Chapter, ChapterDetails, HomeSection, LanguageCode, Manga, MangaStatus, MangaTile, PagedResults } from "paperback-extensions-common";
+import { PagedSearchResults } from "./WebtoonHelperStructures";
+const WEBTOON_DOMAIN = 'https://www.webtoons.com'
+
+export const parseMangaDetailsChallenge = async ($: CheerioStatic, mangaId: string): Promise<Manga> => {
+const title = $('h3._challengeTitle').text().replace('DASHBOARD', '').trim()
+
+const image = $('img', $('span.thmb')).attr('src')
+
+if(!image) {
+    throw new Error(`Failed to parse MangaDetails challenge image for ${mangaId}`)
+}
+
+let rating = $('em#_starScoreAverage').text()
+const completed = $('span.txt_ico_completed2') === undefined ? MangaStatus.ONGOING : MangaStatus.COMPLETED
+const author = $('span.author').text().replace(/author info/, '').trim()
+const description = $('p.summary').text().trim()
+
+if(isNaN(Number(rating))) {
+    rating = '0'
+}
+
+return createManga({
+    id: mangaId,
+    titles: [title],
+    image: image,
+    rating: Number(rating),
+    status: completed,
+    author: author,
+    desc: description
+})
+}
+
+export const parseMangaDetailsOrig = async ($: CheerioStatic, mangaId: string): Promise<Manga> => {
+    const title = $("h1.subj").text();
+
+    // The image is awkwardly embedded into a style, parse that out
+    const imageContext = $("div.detail_body").attr("style");
+    const image = imageContext?.match(/url\((.+)\)/);
+
+    if (!imageContext || !image || !image[1]) {
+      throw new Error(`Failed to parse MangaDetails for ${mangaId}`);
+    }
+
+    let rating = $("em#_starScoreAverage").text();
+    const completed = $("span.txt_ico_completed2") === undefined ? MangaStatus.ONGOING : MangaStatus.COMPLETED;
```

Comment:
```suggestion
    const completed = $("p.day_info").text().includes('COMPLETED') ? MangaStatus.COMPLETED : MangaStatus.ONGOING;
```
Tested and this works